Unable to parse '2000-01-01T00:00:00' as timestamp

For `mss-crm-contact` (Dynamics CRM) it seems like there are some timestamps that we are not able to parse

```
zio.FiberFailure: Text '2000-01-01T00:00:00' could not be parsed at index 19
	at java.base/java.time.format.DateTimeFormatter.parseResolved0(Unknown Source)
	at java.base/java.time.format.DateTimeFormatter.parse(Unknown Source)
	at java.base/java.time.LocalDateTime.parse(Unknown Source)
	at com.sneaksanddata.arcane.framework.services.synapse.base.SynapseLinkReader.valueAsTimeStamp(SynapseLinkReader.scala:154)
	at com.sneaksanddata.arcane.framework.services.synapse.base.SynapseLinkReader.valueAsJava(SynapseLinkReader.scala:117)
	at com.sneaksanddata.arcane.framework.services.synapse.base.SynapseLinkReader.convertCell(SynapseLinkReader.scala:109)
	at com.sneaksanddata.arcane.framework.services.synapse.base.SynapseLinkReader.convertRow$$anonfun$1(SynapseLinkReader.scala:104)
	at scala.collection.immutable.List.map(List.scala:251)
	at com.sneaksanddata.arcane.framework.services.synapse.base.SynapseLinkReader.convertRow(SynapseLinkReader.scala:104)
	at com.sneaksanddata.arcane.framework.services.synapse.base.SynapseLinkReader.getChanges$$anonfun$3(SynapseLinkReader.scala:96)
	at zio.Chunk.mapChunk(Chunk.scala:1062)
	at zio.ChunkLike.map(ChunkLike.scala:121)
	at zio.ChunkLike.map$(ChunkLike.scala:39)
	at zio.Chunk.map(Chunk.scala:43)
	at zio.stream.ZStream.map$$anonfun$1(ZStream.scala:1807)
	at zio.stream.ZChannel.reader$lzyINIT8$1$$anonfun$1(ZChannel.scala:620)
	at zio.stream.internal.ChannelExecutor.run$$anonfun$5(ChannelExecutor.scala:194)
	at zio.stream.internal.ChannelExecutor$.readCh$1(ChannelExecutor.scala:843)
	at zio.stream.internal.ChannelExecutor$.readChAux0$1(ChannelExecutor.scala:831)
	at zio.stream.internal.ChannelExecutor$.readCh$1$$anonfun$8(ChannelExecutor.scala:884)
	at zio.stream.ZChannel.$times$greater$$anonfun$1(ZChannel.scala:78)
	at zio.stream.internal.ChannelExecutor.doneSucceed(ChannelExecutor.scala:367)
	at zio.stream.internal.ChannelExecutor.run$$anonfun$2(ChannelExecutor.scala:165)
Caused by: java.time.format.DateTimeParseException: Text '2000-01-01T00:00:00' could not be parsed at index 19
	at java.base/java.time.format.DateTimeFormatter.parseResolved0(Unknown Source)
	at java.base/java.time.format.DateTimeFormatter.parse(Unknown Source)
	at java.base/java.time.LocalDateTime.parse(Unknown Source)
	at com.sneaksanddata.arcane.framework.services.synapse.base.SynapseLinkReader.valueAsTimeStamp(SynapseLinkReader.scala:154)
	at com.sneaksanddata.arcane.framework.services.synapse.base.SynapseLinkReader.valueAsJava(SynapseLinkReader.scala:117)
	at com.sneaksanddata.arcane.framework.services.synapse.base.SynapseLinkReader.convertCell(SynapseLinkReader.scala:109)
	at com.sneaksanddata.arcane.framework.services.synapse.base.SynapseLinkReader.convertRow$$anonfun$1(SynapseLinkReader.scala:104)
	at scala.collection.immutable.List.map(List.scala:251)
	at com.sneaksanddata.arcane.framework.services.synapse.base.SynapseLinkReader.convertRow(SynapseLinkReader.scala:104)
	at com.sneaksanddata.arcane.framework.services.synapse.base.SynapseLinkReader.getChanges$$anonfun$3(SynapseLinkReader.scala:96)
	at zio.Chunk.mapChunk(Chunk.scala:1062)
	at zio.ChunkLike.map(ChunkLike.scala:121)
	at zio.ChunkLike.map$(ChunkLike.scala:39)
	at zio.Chunk.map(Chunk.scala:43)
	at zio.stream.ZStream.map$$anonfun$1(ZStream.scala:1807)
	at zio.stream.ZChannel.reader$lzyINIT8$1$$anonfun$1(ZChannel.scala:620)
	at zio.stream.internal.ChannelExecutor.run$$anonfun$5(ChannelExecutor.scala:194)
	at zio.stream.internal.ChannelExecutor$.readCh$1(ChannelExecutor.scala:843)
	at zio.stream.internal.ChannelExecutor$.readChAux0$1(ChannelExecutor.scala:831)
	at zio.stream.internal.ChannelExecutor$.readCh$1$$anonfun$8(ChannelExecutor.scala:884)
	at zio.stream.ZChannel.$times$greater$$anonfun$1(ZChannel.scala:78)
	at zio.stream.internal.ChannelExecutor.doneSucceed(ChannelExecutor.scala:367)
	at zio.stream.internal.ChannelExecutor.run$$anonfun$2(ChannelExecutor.scala:165)
	at zio.internal.FiberRuntime.runLoop(FiberRuntime.scala:1103)
	at zio.internal.FiberRuntime.runLoop(FiberRuntime.scala:1094)
	at zio.internal.FiberRuntime.runLoop(FiberRuntime.scala:1094)
	at zio.internal.FiberRuntime.evaluateEffect(FiberRuntime.scala:412)
	at zio.internal.FiberRuntime.evaluateMessageWhileSuspended(FiberRuntime.scala:487)
	at zio.internal.FiberRuntime.drainQueueOnCurrentThread(FiberRuntime.scala:249)
	at zio.internal.FiberRuntime.run(FiberRuntime.scala:137)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
	Suppressed: zio.Cause$FiberTrace: Exception in thread "zio-fiber-" java.time.format.DateTimeParseException: Text '2000-01-01T00:00:00' could not be parsed at index 19
```

--- a/build.sbt
+++ b/build.sbt
@@ -25,7 +25,7 @@ lazy val plugin = (project in file("."))
     name := "arcane-stream-microsoft-synapse-link",
     idePackagePrefix := Some("com.sneaksanddata.arcane.microsoft_synapse_link"),
 
-    libraryDependencies += "com.sneaksanddata" % "arcane-framework_3" % "0.9.7",
+    libraryDependencies += "com.sneaksanddata" % "arcane-framework_3" % "1.0.1",
     libraryDependencies += "io.netty" % "netty-tcnative-boringssl-static" % "2.0.65.Final",
 
     // bugfix for upgrade header

--- a/docker-compose.yaml
+++ b/docker-compose.yaml
@@ -68,7 +68,7 @@ services:
 ##    volumes:
 ##      - ./http-proxy-envoy-config.yaml:/etc/envoy/envoy.yaml
   lakekeeper:
-    image: quay.io/lakekeeper/catalog:latest-main
+    image: quay.io/lakekeeper/catalog:v0.8.5
     network_mode: host
     environment:
 #      - LAKEKEEPER__PG_ENCRYPTION_KEY=This-is-NOT-Secure!
@@ -93,7 +93,7 @@ services:
       minio:
         condition: service_healthy
   lakekeeper_migrate:
-    image: quay.io/lakekeeper/catalog:latest-main
+    image: quay.io/lakekeeper/catalog:v0.8.5
     network_mode: host
     environment:
 #      - LAKEKEEPER__PG_ENCRYPTION_KEY=This-is-NOT-Secure!

--- a/src/main/scala/main.scala
+++ b/src/main/scala/main.scala
@@ -9,39 +9,22 @@ import com.sneaksanddata.arcane.framework.models.app.StreamContext
 import com.sneaksanddata.arcane.framework.models.settings.{GroupingSettings, VersionedDataGraphBuilderSettings}
 import com.sneaksanddata.arcane.framework.services.app.{GenericStreamRunnerService, PosixStreamLifetimeService}
 import com.sneaksanddata.arcane.framework.services.app.base.{StreamLifetimeService, StreamRunnerService}
+import com.sneaksanddata.arcane.framework.services.caching.schema_cache.MutableSchemaCache
 import com.sneaksanddata.arcane.framework.services.synapse.SynapseHookManager
-import com.sneaksanddata.arcane.framework.services.merging.{JdbcMergeServiceClient, MutableSchemaCache}
-import com.sneaksanddata.arcane.framework.services.storage.services.AzureBlobStorageReader
-import com.sneaksanddata.arcane.framework.services.streaming.processors.transformers.{
-  FieldFilteringTransformer,
-  StagingProcessor
-}
+import com.sneaksanddata.arcane.framework.services.merging.JdbcMergeServiceClient
+import com.sneaksanddata.arcane.framework.services.streaming.processors.transformers.{FieldFilteringTransformer, StagingProcessor}
 import zio.*
 import zio.logging.backend.SLF4J
-import com.sneaksanddata.arcane.framework.services.filters.{
-  FieldsFilteringService,
-  FieldsFilteringService as FrameworkFieldsFilteringService
-}
-import com.sneaksanddata.arcane.framework.services.lakehouse.IcebergS3CatalogWriter
-import com.sneaksanddata.arcane.framework.services.streaming.data_providers.backfill.{
-  GenericBackfillStreamingMergeDataProvider,
-  GenericBackfillStreamingOverwriteDataProvider
-}
-import com.sneaksanddata.arcane.framework.services.streaming.graph_builders.{
-  GenericGraphBuilderFactory,
-  GenericStreamingGraphBuilder
-}
+import com.sneaksanddata.arcane.framework.services.filters.{FieldsFilteringService, FieldsFilteringService as FrameworkFieldsFilteringService}
+import com.sneaksanddata.arcane.framework.services.iceberg.IcebergS3CatalogWriter
+import com.sneaksanddata.arcane.framework.services.storage.services.azure.AzureBlobStorageReader
+import com.sneaksanddata.arcane.framework.services.streaming.data_providers.backfill.{GenericBackfillStreamingMergeDataProvider, GenericBackfillStreamingOverwriteDataProvider}
+import com.sneaksanddata.arcane.framework.services.streaming.graph_builders.{GenericGraphBuilderFactory, GenericStreamingGraphBuilder}
 import com.sneaksanddata.arcane.framework.services.streaming.processors.GenericGroupingTransformer
 import com.sneaksanddata.arcane.framework.services.streaming.processors.batch_processors.backfill.BackfillApplyBatchProcessor
-import com.sneaksanddata.arcane.framework.services.streaming.processors.batch_processors.streaming.{
-  DisposeBatchProcessor,
-  MergeBatchProcessor
-}
+import com.sneaksanddata.arcane.framework.services.streaming.processors.batch_processors.streaming.{DisposeBatchProcessor, MergeBatchProcessor}
 import com.sneaksanddata.arcane.framework.services.synapse.base.{SynapseLinkDataProvider, SynapseLinkReader}
-import com.sneaksanddata.arcane.framework.services.synapse.{
-  SynapseBackfillOverwriteBatchFactory,
-  SynapseLinkStreamingDataProvider
-}
+import com.sneaksanddata.arcane.framework.services.synapse.{SynapseBackfillOverwriteBatchFactory, SynapseLinkStreamingDataProvider}
 
 object main extends ZIOAppDefault {
 

--- a/src/main/scala/main.scala
+++ b/src/main/scala/main.scala
@@ -12,19 +12,37 @@ import com.sneaksanddata.arcane.framework.services.app.base.{StreamLifetimeServi
 import com.sneaksanddata.arcane.framework.services.caching.schema_cache.MutableSchemaCache
 import com.sneaksanddata.arcane.framework.services.synapse.SynapseHookManager
 import com.sneaksanddata.arcane.framework.services.merging.JdbcMergeServiceClient
-import com.sneaksanddata.arcane.framework.services.streaming.processors.transformers.{FieldFilteringTransformer, StagingProcessor}
+import com.sneaksanddata.arcane.framework.services.streaming.processors.transformers.{
+  FieldFilteringTransformer,
+  StagingProcessor
+}
 import zio.*
 import zio.logging.backend.SLF4J
-import com.sneaksanddata.arcane.framework.services.filters.{FieldsFilteringService, FieldsFilteringService as FrameworkFieldsFilteringService}
+import com.sneaksanddata.arcane.framework.services.filters.{
+  FieldsFilteringService,
+  FieldsFilteringService as FrameworkFieldsFilteringService
+}
 import com.sneaksanddata.arcane.framework.services.iceberg.IcebergS3CatalogWriter
 import com.sneaksanddata.arcane.framework.services.storage.services.azure.AzureBlobStorageReader
-import com.sneaksanddata.arcane.framework.services.streaming.data_providers.backfill.{GenericBackfillStreamingMergeDataProvider, GenericBackfillStreamingOverwriteDataProvider}
-import com.sneaksanddata.arcane.framework.services.streaming.graph_builders.{GenericGraphBuilderFactory, GenericStreamingGraphBuilder}
+import com.sneaksanddata.arcane.framework.services.streaming.data_providers.backfill.{
+  GenericBackfillStreamingMergeDataProvider,
+  GenericBackfillStreamingOverwriteDataProvider
+}
+import com.sneaksanddata.arcane.framework.services.streaming.graph_builders.{
+  GenericGraphBuilderFactory,
+  GenericStreamingGraphBuilder
+}
 import com.sneaksanddata.arcane.framework.services.streaming.processors.GenericGroupingTransformer
 import com.sneaksanddata.arcane.framework.services.streaming.processors.batch_processors.backfill.BackfillApplyBatchProcessor
-import com.sneaksanddata.arcane.framework.services.streaming.processors.batch_processors.streaming.{DisposeBatchProcessor, MergeBatchProcessor}
+import com.sneaksanddata.arcane.framework.services.streaming.processors.batch_processors.streaming.{
+  DisposeBatchProcessor,
+  MergeBatchProcessor
+}
 import com.sneaksanddata.arcane.framework.services.synapse.base.{SynapseLinkDataProvider, SynapseLinkReader}
-import com.sneaksanddata.arcane.framework.services.synapse.{SynapseBackfillOverwriteBatchFactory, SynapseLinkStreamingDataProvider}
+import com.sneaksanddata.arcane.framework.services.synapse.{
+  SynapseBackfillOverwriteBatchFactory,
+  SynapseLinkStreamingDataProvider
+}
 
 object main extends ZIOAppDefault {
 

--- a/src/main/scala/models/app/MicrosoftSynapseLinkStreamContext.scala
+++ b/src/main/scala/models/app/MicrosoftSynapseLinkStreamContext.scala
@@ -5,28 +5,9 @@ import models.app.contracts.StreamSpec
 
 import com.sneaksanddata.arcane.framework.models.app.StreamContext
 import com.sneaksanddata.arcane.framework.models.settings
-import com.sneaksanddata.arcane.framework.models.settings.{
-  BackfillBehavior,
-  BackfillSettings,
-  BufferingStrategy,
-  FieldSelectionRule,
-  FieldSelectionRuleSettings,
-  GroupingSettings,
-  OptimizeSettings,
-  OrphanFilesExpirationSettings,
-  SnapshotExpirationSettings,
-  SourceBufferingSettings,
-  StagingDataSettings,
-  SynapseSourceSettings,
-  TableFormat,
-  TableMaintenanceSettings,
-  TablePropertiesSettings,
-  TargetTableSettings,
-  VersionedDataGraphBuilderSettings
-}
-import com.sneaksanddata.arcane.framework.services.lakehouse.IcebergCatalogCredential
-import com.sneaksanddata.arcane.framework.services.lakehouse.base.{IcebergCatalogSettings, S3CatalogFileIO}
-import com.sneaksanddata.arcane.framework.services.merging.JdbcMergeServiceClientOptions
+import com.sneaksanddata.arcane.framework.models.settings.{BackfillBehavior, BackfillSettings, BufferingStrategy, FieldSelectionRule, FieldSelectionRuleSettings, GroupingSettings, IcebergCatalogSettings, JdbcMergeServiceClientSettings, OptimizeSettings, OrphanFilesExpirationSettings, SnapshotExpirationSettings, SourceBufferingSettings, StagingDataSettings, SynapseSourceSettings, TableFormat, TableMaintenanceSettings, TablePropertiesSettings, TargetTableSettings, VersionedDataGraphBuilderSettings}
+import com.sneaksanddata.arcane.framework.services.iceberg.IcebergCatalogCredential
+import com.sneaksanddata.arcane.framework.services.iceberg.base.S3CatalogFileIO
 import zio.ZLayer
 
 import java.time.format.DateTimeFormatter
@@ -55,7 +36,7 @@ case class MicrosoftSynapseLinkStreamContext(spec: StreamSpec)
     extends StreamContext
     with GroupingSettings
     with IcebergCatalogSettings
-    with JdbcMergeServiceClientOptions
+    with JdbcMergeServiceClientSettings
     with VersionedDataGraphBuilderSettings
     with AzureConnectionSettings
     with TargetTableSettings
@@ -174,7 +155,7 @@ case class MicrosoftSynapseLinkStreamContext(spec: StreamSpec)
 object MicrosoftSynapseLinkStreamContext {
 
   type Environment = StreamContext & GroupingSettings & VersionedDataGraphBuilderSettings & IcebergCatalogSettings &
-    JdbcMergeServiceClientOptions & AzureConnectionSettings & TargetTableSettings & MicrosoftSynapseLinkStreamContext &
+    JdbcMergeServiceClientSettings & AzureConnectionSettings & TargetTableSettings & MicrosoftSynapseLinkStreamContext &
     GraphExecutionSettings & TablePropertiesSettings & FieldSelectionRuleSettings & BackfillSettings &
     StagingDataSettings & SynapseSourceSettings & SourceBufferingSettings
 

--- a/src/main/scala/models/app/MicrosoftSynapseLinkStreamContext.scala
+++ b/src/main/scala/models/app/MicrosoftSynapseLinkStreamContext.scala
@@ -5,7 +5,27 @@ import models.app.contracts.StreamSpec
 
 import com.sneaksanddata.arcane.framework.models.app.StreamContext
 import com.sneaksanddata.arcane.framework.models.settings
-import com.sneaksanddata.arcane.framework.models.settings.{BackfillBehavior, BackfillSettings, BufferingStrategy, FieldSelectionRule, FieldSelectionRuleSettings, GroupingSettings, IcebergCatalogSettings, JdbcMergeServiceClientSettings, OptimizeSettings, OrphanFilesExpirationSettings, SnapshotExpirationSettings, SourceBufferingSettings, StagingDataSettings, SynapseSourceSettings, TableFormat, TableMaintenanceSettings, TablePropertiesSettings, TargetTableSettings, VersionedDataGraphBuilderSettings}
+import com.sneaksanddata.arcane.framework.models.settings.{
+  BackfillBehavior,
+  BackfillSettings,
+  BufferingStrategy,
+  FieldSelectionRule,
+  FieldSelectionRuleSettings,
+  GroupingSettings,
+  IcebergCatalogSettings,
+  JdbcMergeServiceClientSettings,
+  OptimizeSettings,
+  OrphanFilesExpirationSettings,
+  SnapshotExpirationSettings,
+  SourceBufferingSettings,
+  StagingDataSettings,
+  SynapseSourceSettings,
+  TableFormat,
+  TableMaintenanceSettings,
+  TablePropertiesSettings,
+  TargetTableSettings,
+  VersionedDataGraphBuilderSettings
+}
 import com.sneaksanddata.arcane.framework.services.iceberg.IcebergCatalogCredential
 import com.sneaksanddata.arcane.framework.services.iceberg.base.S3CatalogFileIO
 import zio.ZLayer

